Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -1085,7 +1085,7 @@ double Element::scroll_top() const
         return window->scroll_y();
 
     // 8. If the element does not have any associated box, return zero and terminate these steps.
-    if (!layout_node() || !is<Layout::Box>(layout_node()))
+    if (!layout_node() || !is<Layout::Box>(layout_node()) || !paintable_box())
```

Comment:
can we instead also stop creating layout node for colgroup?

Reply:
Sure thing, how do I go about doing that?

Reply:
I've tried looking into this, and I must admit, haven't quite figured this out yet. Do you think that will be proper fix, and I should drop this change (i.e every layout node should always have a paintable), and this becomes a `VERIFY` instead?

Reply:
ok, for now, `!paintable_box()` can be replaced with a check, that paintable box is expected to be missing for colgroup layout node specifically, and it will be fine.

Reply:
@kalenikaliaksandr ok - sorry for the delay, i've been busy lately. made those changes, hopefully this is what you meant :^)